### PR TITLE
Make sure commit reference works across repos

### PR DESCRIPTION
**Pull request by Nyholm**
¯\\_(ツ)_/¯

### CODE REVIEW

File: .github/workflows/website.yml [hunk 1]
```diff
@@ -44,5 +44,5 @@ jobs:
         cd website
         git config --local user.email "github@async-aws.com"
         git config --local user.name "AsyncAWS Bot"
-        git commit -m "Update website from commit `git --git-dir=./../aws/.git rev-parse HEAD`" -a
+        git commit -m "Update website from commit async-aws/aws@`git --git-dir=./../aws/.git rev-parse HEAD`" -a
```
Comment (Nyholm) [inline .github/workflows/website.yml]: Should I update this to `github.sha`?

Comment (jderusse) [reply .github/workflows/website.yml]: if it work, yes, it's more readable IMO

File: .github/workflows/static.yml [hunk 2]
```diff
@@ -12,10 +12,6 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
-      - name: Download dependencies
-        run: |
-          composer update --no-interaction --prefer-dist --optimize-autoloader
```
Comment (Nyholm) [inline .github/workflows/static.yml]: There is no need for composer update here. 

See https://github.com/OskarStark/phpstan-ga/blob/master/entrypoint.sh#L8

Comment (Nyholm) [reply .github/workflows/static.yml]: Hm... I was clearly wrong.. strange

Comment (jderusse) [reply .github/workflows/static.yml]: because of no-dev.

but looks like they added a new `$REQUIRE_DEV` variables.

**Pull request merged**